proto/cosmos/base/simulate/v1beta1/simulate.proto

```diff
@@ -0,0 +1,33 @@
+syntax = "proto3";
+package cosmos.base.simulate.v1beta1;
```

Comment:
This shouldn't be in `base`, it should be in `cosmos.tx`

Reply:
Let's fix this when we address #7355. Originally this was in a single tx `Service` in #6872. Ideally those are the same gRPC service. I'm not sure why we would need to separate them.

Reply:
True, this endpoint could definitely go in with those in #7355 

Note: there are [some other TM rpc methods](https://docs.tendermint.com/master/rpc/) which we might or might not want to wrap under gRPC at some point. To avoid having too many services, we could regroup them under one big module-agnostic service, maybe not called `Tx` though. There's also the reflection service, which could also be merged into it.

Reply:
Let's bundle together stuff that is related. Simulate, get tx by hash, etc. all depend on `Tx`. The original `tx` `Service` was intended to cover all that stuff. Reflection doesn't depend on `Tx` so IMHO should remain separate.

Reply:
Right, I was pointing out that our bundling will then be different than TM's bundling (e.g. getTxByHash goes in `Info`, not in `Tx`, see link above), which might be confusing for people, esp. if one day TM has its own gRPC.